gtk4/src/auto/stack_page.rs

```diff
@@ -42,7 +42,7 @@ impl StackPage {
     }
 
     #[doc(alias = "gtk_stack_page_get_needs_attention")]
-    pub fn is_needs_attention(&self) -> bool {
+    pub fn needs_attention(&self) -> bool {
```

Comment:
Nice! :) i really like the new name

Reply:
Thanks :)

Reply:
BTW, this is an automatic conversion on behalf of `fix-getters-rules` which is also automatically detected on the call sites when migrating code using `fix-getters-calls`.